Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION := $(shell git describe --exact-match --tags HEAD)
+VERSION := $(shell git describe --tags HEAD)
```

Comment:
Actually, we wanted this to explicitly fail if the build isn't the tagged commit. We only want the tagged commits to report a valid SemVer value to the backend (this value is going to be piped to the backend through the User-Agent request header). I did notice that the Makefile prints an error when you build on a non-tagged commit. This script could be improved to silence that error when building a non-tagged commit.

Reply:
I'm not sure what that means to the backend, but during development it seems useful to me to have a rough indication of the version (+alpha/beta or other change indicator).  I'd be OK if that is a second field that is just displayed.

I can live with date+SHA on stuff that is not otherwise versioned, here I really want to have the last or next release as an anchor for what is reported.

Reply:
I am not certain that the current commit (26Jun2024) adding -dev still works for backend reporting.